### PR TITLE
Sprint6 axios error

### PR DESCRIPTION
**Pull request by AbdulSir**

### CODE REVIEW

File: frontend/src/router/index.js [hunk 1]
```diff
@@ -101,6 +101,21 @@ const Message = () => import('@/views/apps/email/Message')
 
 Vue.use(Router)
 
+const validPathNames = [
```
Comment (AdrienKamran) [inline frontend/src/router/index.js:104]: This change was most likely carried over from #42 by accident. This has been resolved by @purplesmurf1998 and myself on the previous PR.

**Review by AdrienKamran — CHANGES_REQUESTED** [review]

File: frontend/src/router/index.js (hunk 1 above)
Comment (qandeelarshad) [inline frontend/src/router/index.js:104]: Looks good, except this change from #42 that has been resolved by @purplesmurf1998 and @AdrienKamran on the previous pull request.

**Review by qandeelarshad — CHANGES_REQUESTED** [review]

File: frontend/src/router/index.js (hunk 1 above)
Comment (AbdulSir) [reply frontend/src/router/index.js]: Yes, it seems that it was carried over. I have updated in this [commit](https://github.com/purplesmurf1998/Autotracks/pull/44/commits/f7136c400971f2d84bd6c9d86b98e5c962d513a3)

Comment (AbdulSir) [reply frontend/src/router/index.js]: Already addressed this in Qandeel's reply. Thanks for reviewing the code.

**Review by AdrienKamran — APPROVED** [review]
Changes were discussed earlier today, approved

**Review by qandeelarshad — APPROVED** [review]
These changes have been discussed, approved.

**Pull request merged**